common/templates/options.php

```diff
@@ -77,6 +77,20 @@
                     <?php if (!$plugin->woocommerce): ?>
                     <p class="description"><?php _e('Install and activate WooCommerce to use this functionality.', 'webwinkelkeur'); ?></p>
                     <?php endif; ?>
+                    <fieldset>
+                        <br>
+                        <label for="<?= $plugin->getOptionName('order_status'); ?>"> <?php _e('Select order status', 'webwinkelkeur'); ?>
```

Comment:
* A label that contains an input element automatically applies to that input element, there's no need to use `for`.
* The `for` on a label refers to the `id` on the input element, not the `name`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label